src/main/resources/interface-specification.yml

```diff
@@ -471,45 +571,112 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
-  /status:
-    get:
-      security: []
-      summary: Returns the application status
-      description: |
-        Returns the application status. For testing purposes, it might randomly reply with an error.
-      operationId: get_status
-      tags:
-        - health
-      responses:
-        '200':
-          description: |
-            This is the valid status from the server.
-          content:
-            application/problem+json:
-              schema:
-                $ref: '#/components/schemas/Problem'
-  '/agreements/{agreementId}/purposes':
-    get:
+  '/organizations/origin/{origin}/externalId/{externalId}/attributes/{code}':
+    post:
       tags:
         - gateway
-      summary: Get an purpose
-      description: Retrieve purposes of the agreement
-      operationId: getAgreementPurposes
+      summary: Associate an organization with an attribute
+      description: Associate an organization with an attribute
+      operationId: bindOrganizationAttribute
+      security:
+        - bearerAuth: [ ]
       parameters:
-        - name: agreementId
-          in: path
-          description: the agreement identifier
+        - in: path
+          name: origin
+          schema:
+            type: string
           required: true
+        - in: path
+          name: externalId
           schema:
             type: string
-            format: uuid
-      responses:
-        '200':
-          description: Purposes retrieved
+          required: true
+        - in: path
+          name: code
+          schema:
+            type: string
+          required: true
```

Comment:
I would add parameters descriptions here, as it may be confusing for an external view point